manifests/init.pp

```diff
@@ -37,6 +37,9 @@
 #
 # $package_names::      Packages that this module ensures are present instead of the default
 #
+# $enable_ostree::      Boolean to enable ostree plugin. This requires existence of a ostree install.Defaults to false
```

Comment:
I think the defaults make sense here though. Because we have other modules calling this to pass parameters, its useful to know what the default is.